email_processing.py

```
@@ -0,0 +1,197 @@
+        If email was not processed correctly or file was not found in API or server cannot be reached -> return 1
+    '''
+    access_token = None
+    all_files = []
+    # ------------- START config
+#   Created environment variables according to task description:
+    ROSSUM_USER = os.environ.get('ROSSUM_USER')
+    ROSSUM_PASSWORD = os.environ.get('ROSSUM_PASSWORD')
+    FILE_NAME = os.environ.get('FILE_NAME')
+    API_URL = os.environ.get('API_URL')
+    EMAIL = os.environ.get('EMAIL')
+
+    # My own variables used for authentication at smtp.gmail.com to send an email with attachment
+    # Needs to have configured gmail account (gmail smtp is used in import_by_email function) to send email to EMAIL address of trial organization
+    SENDER_EMAIL_ADDRESS = os.environ.get('SENDER_EMAIL_ADDRESS')
+    SENDER_EMAIL_PASSWORD = os.environ.get('SENDER_EMAIL_PASSWORD')
+
+    # -------------- END config
+
+    def get_login_data(self):
+        return {
+            'username': self.ROSSUM_USER,
+            'password': self.ROSSUM_PASSWORD,
+        }
+
+    def get_access_token(self):
+        user = self.ROSSUM_USER
+
+        login_api_url = f'https://{self.API_URL}/v1/auth/login'
+        login_data = self.get_login_data()
+
+        print('Performing authentication at RossumAPI ... ')
+        try:
+            r = requests.post(login_api_url, data=login_data)
+            # Get HTTP status code of a domain + path + data
+            if r.status_code not in range(200, 299):
+                raise Exception("Could not authenticate client.")
+                return False
+
+            response_from_elis = r.json()
+            token = response_from_elis['key']
+            self.access_token = token
+            print(f"User {user} has been successfully authenticated.")
+            return token
+        except requests.exceptions.RequestException:
+            raise Exception(f'Failed to connect to {login_api_url}') from None
```

I don't think it is necessary to redress the `RequestException`. I'd just let it be raised without catching it in the `try ... except ...`.